partials/examples/rule.html

```diff
@@ -0,0 +1,7 @@
+<p>
+    This example shows the build in rules.
+    The rule is applied to the name field and the birthDate field. If the selected nationality is 'DE' then the name field is hidden and the birthDate field is shown.
```

Comment:
I'd change this sentence slightly: In this example, a single rule is applied to the `name` and the `birthDate` field, depending on the value of the `nationality`. If the selected nationality is 'DE' then the `name` field is hidden and the `birthDate` field is shown